staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -210,20 +214,21 @@ type SecureServingInfo struct {
 // NewConfig returns a Config struct with the default values
 func NewConfig(codecs serializer.CodecFactory) *Config {
 	return &Config{
-		Serializer:                   codecs,
-		ReadWritePort:                443,
-		RequestContextMapper:         apirequest.NewRequestContextMapper(),
-		BuildHandlerChainFunc:        DefaultBuildHandlerChain,
-		LegacyAPIGroupPrefixes:       sets.NewString(DefaultLegacyAPIPrefix),
-		DisabledPostStartHooks:       sets.NewString(),
-		HealthzChecks:                []healthz.HealthzChecker{healthz.PingHealthz},
-		EnableIndex:                  true,
-		EnableDiscovery:              true,
-		EnableProfiling:              true,
-		MaxRequestsInFlight:          400,
-		MaxMutatingRequestsInFlight:  200,
-		MinRequestTimeout:            1800,
-		EnableAPIResponseCompression: utilfeature.DefaultFeatureGate.Enabled(features.APIResponseCompression),
+		Serializer:                    codecs,
+		ReadWritePort:                 443,
+		RequestContextMapper:          apirequest.NewRequestContextMapper(),
+		BuildHandlerChainFunc:         DefaultBuildHandlerChain,
+		LegacyAPIGroupPrefixes:        sets.NewString(DefaultLegacyAPIPrefix),
+		DisabledPostStartHooks:        sets.NewString(),
+		HealthzChecks:                 []healthz.HealthzChecker{healthz.PingHealthz},
+		EnableIndex:                   true,
+		EnableDiscovery:               true,
+		EnableProfiling:               true,
+		MaxRequestsInFlight:           400,
+		MaxMutatingRequestsInFlight:   200,
+		MinRequestTimeout:             1800,
+		DisconnectChanceOnMaxRequests: 0.01,
```

Comment:
That is not the default, it's the default for any given generic apiserver (many of which are run on kubernetes and will scale up and down, like example apiserver).  It's a better default than 0 for that scenario because extension API servers are expected to be run under deployments.